configs/nuimages/metafile.yml

```diff
@@ -8,9 +8,16 @@ Collections:
         - RoI Align
         - RPN
     Paper:
-      - https://arxiv.org/abs/1703.06870
-      - https://arxiv.org/abs/1712.00726v1
+      URL:
+        - https://arxiv.org/abs/1703.06870
```

Comment:
Since it is a dataset, we also do not need to write a paper URL and code for that.